src/inspirehep-holdingpen-js/templates/inspirehep-holdingpen-js/details/details_decision_hep.html

```diff
@@ -14,17 +14,26 @@
           ng-click="Utils.setDecision('accept')">Accept
   </button>
 
-  <button ng-if="vm.record.metadata.acquisition_source.method == 'submitter'"
-          ng-class="{'btn btn-danger': vm.record._extra_data.journal_coverage == 'partial',
-          'btn btn-danger-weak': vm.record._extra_data.journal_coverage == 'full'}"
+  <button ng-if="vm.record.metadata.acquisition_source.method == 'submitter' && vm.record._extra_data.journal_coverage == 'full'"
+          class="btn btn-danger-weak"
           ng-click="Utils.setRejectionReason()"
           uib-tooltip="The article belongs to a fully taken journal">Reject
   </button>
-  <button ng-if="vm.record.metadata.acquisition_source.method == 'hepcrawl'"
-          class="btn btn-danger"
+    <button ng-if="vm.record.metadata.acquisition_source.method == 'submitter'
+                   && (!vm.record._extra_data.journal_coverage || vm.record._extra_data.journal_coverage == 'partial')"
```

Comment:
@harunurhan is the layout ok for this long condition in the ng-if?

Reply:
yeah, OK I'd put these long stuff in js file with descriptive function names that tells us what these condition means, it's like this everywhere in the project unfortunately :( So you don't have to do it like this for now,